docs/de/marketing_guide_to_firefox_l10n.md

```diff
@@ -0,0 +1,216 @@
+# How to Firefox in German
+
+Dieses Dokument soll die Erstellung und Lokalisierung von Texten erleichtern – für optimal aufbereitete und konsistente Inhalte. Wichtig hierfür sind:
+
+* [eine zielgruppengerechte und markenspezifische Ansprache](Persönlichkeit & Tonalität)
```

Comment:
Are these two supposed to be links? Then you need to link to `#pers%C3%B6nlichkeit--tonalit%C3%A4t` and `#glossar`. You can find the anchor by looking at [the page](https://github.com/mozilla-l10n/styleguides/blob/bae86d9d1c9775a36477ac98dee855695604213d/docs/de/marketing_guide_to_firefox_l10n.md) and copying the last part of the link.